controllers/storagecluster/external_resources.go

```diff
@@ -189,7 +189,11 @@ func newExternalGatewaySpec(rgwEndpoint string, reqLogger logr.Logger, tlsEnable
 		reqLogger.Error(err, "Host IP should not be empty in rgw endpoint")
 		return nil, err
 	}
+	address := net.ParseIP(hostIP)
 	gateWay.ExternalRgwEndpoints = []cephv1.EndpointAddress{{IP: hostIP}}
+	if address == nil {
```

Comment:
This doesn't change the behavior, but there is no need for the `address` var
```suggestion
	if net.ParseIP(hostIP) == nil {
```

Reply:
Updated